interfaces/builtin/mir_test.go

```diff
@@ -0,0 +1,95 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
```

Comment:
ok, zyga/jamie - i think it's probably ready for another round, spend large amount of today/y'day cleaning up